index.html

```diff
@@ -135,9 +135,9 @@ <h2 style="clear:both">Photo Album <a class="float-end btn btn-outline-primary"
     // The summer schedule for RCR applies from Memorial Day to Labor Day.
     // Memorial Day is the last Monday in May, Labor Day is the first Monday in September.
     const today = new Date();
-    const memorialDay = new Date(today.getFullYear() + 1, 4, 31);
+    const memorialDay = new Date(today.getFullYear(), 4, 31);
     memorialDay.setDate(memorialDay.getDate() - (memorialDay.getDay() + 6) % 7);
-    const laborDay = new Date(today.getFullYear() + 1, 8, 1);
```

Comment:
We were checking if today's date fell between _next_ year's Memorial Day and Labor Day, which will always be false.